packages/experimental/Expander/windows/ReactNativeExpander/ExpanderView.idl

```diff
@@ -0,0 +1,9 @@
+namespace ReactNativeExpander{
+
+    [default_interface]
+    runtimeclass ExpanderView : Windows.UI.Xaml.Controls.ContentPresenter {
```

Comment:
We should add a comment to the IDL to explain why we need a ContentPresenter in the first place